From 91ab6b4feda09e2ded45f716d8f283ced06971ad Mon Sep 17 00:00:00 2001
From: Milian Wolff <milian.wolff@kdab.com>
Date: Thu, 25 Jan 2018 14:05:37 +0100
Subject: [PATCH] Properly restore size of KDE5 file picker dialog

The event filter we installed prevented the show event from reaching
the KDEPlatformFileDialogHelper::show method. This prevented the
dialog from being restored to the size chosen by the user previously.

Change-Id: I7af457f19f59d27bd104f8f70d9f4767da6641ee
Reviewed-on: https://gerrit.libreoffice.org/48591
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Thorsten Behrens <Thorsten.Behrens@CIB.de>
---
 vcl/unx/gtk3_kde5/kde5_filepicker.cxx | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vcl/unx/gtk3_kde5/kde5_filepicker.cxx b/vcl/unx/gtk3_kde5/kde5_filepicker.cxx
index 9b3f21db92e9..d774ebc2834e 100644
--- a/vcl/unx/gtk3_kde5/kde5_filepicker.cxx
+++ b/vcl/unx/gtk3_kde5/kde5_filepicker.cxx
@@ -235,7 +235,6 @@ bool KDE5FilePicker::eventFilter(QObject* o, QEvent* e)
             KWindowSystem::setMainWindow(w, _winId);
             if (auto* fileWidget = w->findChild<KFileWidget*>({}, Qt::FindDirectChildrenOnly))
                 fileWidget->setCustomWidget(_extraControls);
-            return false;
         }
     }
     return QObject::eventFilter(o, e);
